scss/shellbar/_search-input.scss

```diff
@@ -9,43 +9,40 @@ $block-search-input: #{$fd-namespace}-search-input;
 $fd-search-closed-conrol-transition: all $fd-animation--speed !default;
 
 .#{$block-search-input} {
-    @include fd-reset();
     display: inline-block;
     z-index: map-get($fd-z-index-levels, "first");
     transition: $fd-search-closed-conrol-transition;
     @include fd-rtl() {
         right: auto;
         left: 0;
     }
-    .#{$fd-namespace}-input-group {
-        .#{$fd-namespace}-input-group__addon {
-            border-color: transparent;
-            @include fd-var-color("background-color", fd-color("shell", 1), --fd-color-shell-1);
-        }
-        .#{$fd-namespace}-input {
-            border-style: solid;
-            border-width: 1px;
-            @include fd-var-color("border-color", fd-color("shell", 2), --fd-color-shell-2);
-            @include fd-var-color("background-color", fd-color("shell", 1), --fd-color-shell-1);
-            @include fd-var-color("color", fd-color("shell",2), --fd-color-shell-2);
-            transition: border-color .125s;
-            padding-left: 12px;
-            padding-right: 12px;
+    .#{$fd-namespace}-input-group__addon--button {
+        border-color: transparent;
+        @include fd-var-color("background-color", fd-color("shell", 1), --fd-color-shell-1);
+        position: relative;
+        &[aria-expanded="true"] {
+            z-index: map-get($fd-z-index-levels, "second");
             &:focus {
                 box-shadow: none;
             }
-            &[aria-hidden="true"],
-            &.is-hidden {
-            opacity: 0;
-            visibility: hidden;
-            }
-            > .fd-shellbar__button[aria-expanded="true"] {
-                position: relative;
-                z-index: map-get($fd-z-index-levels, "second");
-                &:focus {
-                    box-shadow: none;
-                }
-            }
+        }
+    }
+    .#{$fd-namespace}-input {
+        border-style: solid;
+        border-width: 1px;
+        @include fd-var-color("border-color", fd-color("shell", 2), --fd-color-shell-2);
+        @include fd-var-color("background-color", fd-color("shell", 1), --fd-color-shell-1);
+        @include fd-var-color("color", fd-color("shell",2), --fd-color-shell-2);
+        transition: border-color .125s;
+        padding-left: 12px;
+        padding-right: 12px;
+        &:focus {
+            box-shadow: none;
+        }
+        &[aria-hidden="true"],
+        &.is-hidden {
+        opacity: 0;
+        visibility: hidden;
```

Comment:
Not a big fan of this right now, but I have a feeling we will be iterating on shellbar a few times.  I think there are some further optimizations we can achieve, but this is probably okay for now.